ec2api/api/image.py

```diff
@@ -521,23 +521,21 @@ def _format_image(context, image, os_image, images_dict, ids_dict,
                 items_by_os_id=images_dict, ids_by_os_id=ids_dict)
 
     name = os_image.name
-    img_loc = os_image.properties.get('image_location')
```

Comment:
Can we comment out this code as of now, and not delete it? This would most probably be needed to be shown in the future releases.